From jrm8005@gmail.com  Wed Nov 12 14:25:47 2008
From: "J.R. Mauro" <jrm8005@gmail.com>
Date: Fri, 31 Oct 2008 18:58:30 -0400
Subject: Staging: sxg: update README
To: Greg Kroah-Hartman <gregkh@suse.de>
Message-ID: <20081031225830.GA7326@tuxbook-pro.zma.zoran.com>
Content-Disposition: inline


Update readme in drivers/staging/sxg

Signed-off by: J.R. Mauro <jrm8005@gmail.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/staging/sxg/README |    2 --
 1 file changed, 2 deletions(-)

--- a/drivers/staging/sxg/README
+++ b/drivers/staging/sxg/README
@@ -2,8 +2,6 @@ This is the rough cut at a driver for th
 Non-Accelerated 10Gbe network driver.
 
 TODO:
-	- lindent the code
-	- remove typedefs
 	- remove wrappers
 	- checkpatch.pl cleanups
 	- new functionality that the card needs
